<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>songs_edit_screen</class>
 <widget class="QWidget" name="songs_edit_screen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>934</width>
    <height>417</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>90</y>
     <width>251</width>
     <height>276</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="mainLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <item>
     <widget class="QLabel" name="title_l">
      <property name="text">
       <string>Title</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="title_le"/>
    </item>
    <item>
     <widget class="QLabel" name="author_l">
      <property name="text">
       <string>Author</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="author_sle" native="true"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="filenameLayout">
      <item>
       <widget class="QLabel" name="filename_l">
        <property name="text">
         <string>Filename</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="filename_pb">
        <property name="text">
         <string>Select File</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLineEdit" name="filename_le"/>
    </item>
    <item>
     <widget class="QLabel" name="seeableness_l">
      <property name="text">
       <string>Visibility</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="seeablenessLayout">
      <item>
       <widget class="QRadioButton" name="private_rb">
        <property name="text">
         <string>Private</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="public_rb">
        <property name="text">
         <string>Public</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="edit_pb">
      <property name="text">
       <string>Edit</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="userOutput_l">
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancel_pb">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
